### DISCUSSION TITLE
[docs] How to integrate with tools like Postcss and Tailwind

### DISCUSSION THREAD

**Discussion by willsoto** [stitchesjs/stitches#771] [Ideas]
**Is your feature request related to a problem? Please describe.**

I would like to know how to integrate this tool with things like Postcss and Tailwind.

**Describe the solution you'd like**

It could be great to have an "Integrations" section or something similar so people understood the best way to integrate with other popular tools in the ecosystem

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

keywords: postcss, tailwind, autoprefixer

**Comment by peduarte** [comment]
@willsoto hey! I dont understand, sorry. Can you explain what you mean by "integrate"?

**Comment by willsoto** [comment]
@peduarte Sure. The reason I ask at all is because I see [several Tailwind threads](https://github.com/modulz/stitches/issues?q=is%3Aissue+tailwind+is%3Aclosed) each implying that there _was_ some package that integrated Stitches with Tailwind. Based on that, I assume it's not as straightforward as configuring postcss?

Asked another way: If I configure Webpack/Postcss/Tailwind according to their documentation, can I do something like this with Stitches:

```ts
const Button = styled('button', {
  '@apply': 'flex more tailwind properties', // Will this part be processed correctly?
});
```

**Comment by peduarte** [comment]
@willsoto ooh, yes. That was an old feature of Stitches that has been deprecated. We no longer support a tailwind integration. Sorry for the confusion.

**Comment by willsoto** [comment]
@peduarte Sorry, I feel like my question is getting muddled from the context. I understand the integration is deprecated and that is fine. 

I am wondering how I can use tools like Postcss with this library, is it possible?

**Comment by peduarte** [comment]
@willsoto Ok, I've re-opened the issue. I'm sorry I still dont follow what youre trying to achieve. 

"Using Stitches with Postcss" is a bit too vague for me to understand what youre trying to accomplish. 

Think of Stitches as an alternative to styled-component or emotion. It's a CSS in JS library. It has its own syntax for things that arent supported in CSS yet, like abstracting media queries, variants, etc. We then convert that to CSS that browsers understand. 

If you're after something specific, please dont hesitate to ask! One of the maintainers of Stitches, @jonathantneal , is also a maintainer of postcss and various of its plugins

**Comment by willsoto** [comment]
Ok let me ask a different question: If I setup Postcss with Autoprefixer, will the CSS I write via Stitches be prefixed correctly when it gets built?

**Comment by peduarte** [comment]
@willsoto only during server-side rendering. [Example here](https://stitches.dev/docs/server-side-rendering)

**Comment by willsoto** [comment]
I saw that, thank you. 

So are there any plans to make this possible if you are _not_ doing SSR? As an example, [astroturf](https://github.com/4Catalyzer/astroturf) has a special Webpack loader which I assume is what allows it to integrate with that whole toolchain outside of SSR.

**Comment by jonathantneal** [comment]
I’m moving this over to a discussion, since we can’t necessarily take action from within this repo.
